tutorials/preprocessing/plot_55_setting_eeg_reference.py

```diff
@@ -188,6 +191,25 @@
     fig.subplots_adjust(top=0.9)
     fig.suptitle('{} reference'.format(title), size='xx-large', weight='bold')
 
+###############################################################################
+# Using a Bipolar reference
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# To create a Bipolar reference, you can use :meth:`~mne.set_bipolar_reference`
+# along with the respective channel names for ``anode`` and ``cathode`` which
+# creates a new virtual channel that takes the difference between two
+# specified channels (anode and cathode) and drops the original channels by
+# default. The new virtual channel will be annotated with the channel info of
+# the anode with locations set to (0, 0, 0) and coil types set to
+# ``EEG_BIPOLAR``by default. Here we use a contralateral/transverse bipolar
+# reference between channels ``EEG 054`` and ``EEG 055`` as described in
+# :footcite:`YaoEtAl2019` which creates a new virtual channel
+# named ``EEG 054-EEG 055``.
+
+raw_bip_ref = mne.set_bipolar_reference(raw.copy(), anode=['EEG 054'],
+                                        cathode=['EEG 055'])
```

Comment:
`set_bipolar_reference` created a copy by default (see its `copy` param)
```suggestion
raw_bip_ref = mne.set_bipolar_reference(raw, anode=['EEG 054'],
                                        cathode=['EEG 055'])
```